src/Adapter/Product/Combination/QueryHandler/GetEditableCombinationsListHandler.php

```diff
@@ -188,7 +188,8 @@ private function formatEditableCombinationsForListing(
                 (bool) $combination['default_on'],
                 $impactOnPrice,
                 (int) $this->stockAvailableRepository->getForCombination(new CombinationId($combinationId))->quantity,
-                $imagePath
+                $imagePath,
+                (float) $combination['ecotax']
```

Comment:
As suggested by @zuk3975 you'd better use DecimalNumber instead of flat values here